src/js/angular/utils/yasgui-utils.js

```diff
@@ -8,11 +8,42 @@ export const toYasguiOutputModel = ($event) => {
             return buildDownloadAsModel(eventData);
         case EventDataType.NOTIFICATION_MESSAGE:
             return buildNotificationMessageModel(eventData);
+        case EventDataType.COUNT_QUERY:
+            return buildCountQueryModel(eventData);
+        case EventDataType.COUNT_QUERY_RESPONSE:
+            return buildCountQueryResponseModel(eventData);
+        case EventDataType.QUERY:
+            return buildQueryModel(eventData);
         default:
             return eventData;
     }
 };
 
+export const buildCountQueryResponseModel = (eventData) => {
```

Comment:
These function expressions should be declared before their use, e.g. before the toYasguiOutputModel function.